src/core/renderers/canvas/CanvasRenderer.js

```diff
@@ -102,7 +102,11 @@ utils.pluginTarget.mixin(CanvasRenderer);
 /**
  * Renders the object to this canvas view
  *
- * @param object {PIXI.DisplayObject} the object to be rendered
+ * @param displayObject {PIXI.DisplayObject} The object to be rendered
+ * @param [renderTexture] {PIXI.RenderTexture} A render texture to be rendered to. If unset, it will render to the root context.
+ * @param [clear=false] {boolean} Whether to clear the canvas before drawing
+ * @param [transform] {PIXI.Transform} A transformation to be applied
+ * @param skipUpdateTransform {boolean} Whether to skip the update transform
```

Comment:
this is optional, defaulting to `false` (technically undefined, but false in docs is fine)